x-pack/plugins/synthetics/server/synthetics_service/project_monitor/normalizers/common_fields.ts

```diff
@@ -143,6 +143,16 @@ export const getMultipleUrlsOrHostsError = (
   )} per monitor. You monitor was not created or updated.`,
 });
 
+export const getNoUrlsOrHostsError = (
+  monitor: ProjectMonitor,
+  key: 'host' | 'url',
+  version: string
+) => ({
+  id: monitor.id,
+  reason: 'Unsupported Heartbeat option',
+  details: `Monitor must specify 1 ${key} for ${monitor.type} project monitors in ${version}. You monitor was not created or updated.`,
```

Comment:
```suggestion
  details: `${monitor.type} project monitors must have a valid ${key}. Your monitor was not created or updated.`,
```
Suggesting to slightly tweak this:
- Is the version information very important here?
- Is it possible to make acronyms uppercase? URL, HTTP, for example.

Reply:
To provide more context: the validation checks that the user is providing exactly one URL or host. I'm not sure the suggested change makes this clear. It says it's not valid, but it doesn't explain why

Reply:
I think we can write something like

```
`${monitor.type} project monitors must have exactly one ${key}. your monitor was not created or updated.`
```

This message is valid for both cases (no URLs or more than one) and lets us simplify the code.

Reply:
@florent-leborgne I did a small tweak and used the `key` as a literal instead of as a word. Otherwise it could not be a part of the translation. Could you take a final look to confirm is ok?

Reply:
It looks good to me 👍